src/implementations/config/types.ts

```diff
@@ -22,8 +21,19 @@ export type Form = {
   fileName?: string;
 };
 
+export enum NetworkName {
+  local = "local",
+  mainnet = "homestead",
+  goerli = "goerli",
+  sepolia = "sepolia",
+  polygon = "matic",
+  mumbai = "maticmum",
+}
+
```

Comment:
same as `NetworkCmdName`?
can import `NetworkCmdName` as `NetworkName`?

Reply:
ok will see how to reuse it